index.html

```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport"
+      content="width=device-width, initial-scale=1">
+    <title>Works</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+   
+    <!-- container-s -->
+    <div class="container sp">
```

Comment:
The section of your projects cards should be an `<section>` tag, Kindly try to change that